packages/kit/test/apps/basics/test/test.js

```diff
@@ -881,17 +881,21 @@ test.describe('Matchers', () => {
 	test('Matches parameters', async ({ page, clicknav }) => {
 		await page.goto('/routing/matched');
 
-		await clicknav('[href="/routing/matched/a"]');
+		// in SSR, href is relative to the current page!
```

Comment:
Is this expected?

Reply:
Seems like yes: https://github.com/jycouet/kit/blob/64e96dc81d02d703fad7fe055f9c4ff3faeb7808/packages/kit/src/runtime/server/page/render.js#L97-L98

Okay